### PR TITLE
Update migration doc

### PR DESCRIPTION
**Pull request by nporter-adbe**
Minor updates to the migration doc.

### CODE REVIEW

**Comment by sbenedicadb** [comment]
general: it's best to write our documentation and guides without using personal or possessive pronouns.  if possible, think of an alternate way to rewrite the sentence.

**Review by cdhoffmann — APPROVED** [review]

File: Documentation/Migration/Migration_Install.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Migrate an ACPCore iOS app to the latest AEPCore
+
+Existing ACPCore users who are interested in upgrading to the latest AEPCore SDK, this guide describes how to upgrade. We have made upgrading seamless by providing a backward-compatible layer, which will ensure that an existing ACPCore implementation will be compatible with AEPCore.
```
Comment (sbenedicadb) [inline Documentation/Migration/Migration_Install.md:3]: reword the first sentence :)

File: Documentation/Migration/Migration_Install.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Upgrading
+
+### Cocoapods
+
+The easiest way to upgrade from ACPCore to AEPCore is by using Cocoapods. To upgrade, update the reference to `ACPCore` in your projects `Podfile` as such:
```
Comment (sbenedicadb) [inline Documentation/Migration/Migration_Install.md:9]: remove `as such`

File: Documentation/Migration/Migration_Install.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+```diff
+- pod 'ACPCore'
++ pod 'ACPCore', :git => 'https://github.com/adobe/aep-sdk-compatibility-ios.git', :branch => 'main'
+```
+
+ Then run `pod install` and build your project, and you will be using the latest AEPCore SDK. Under the hood, this new podspec brings in the latest AEPCore SDK via Cocoapods.
```
Comment (sbenedicadb) [inline Documentation/Migration/Migration_Install.md:16]: rephrase to use third-person

File: Documentation/Migration/Migration_Install.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+#### Version Control
+
+To control which version of AEPCore will be used when using this upgrade method, specify the versions in the `Podfile`.
+
+```ruby
+pod 'AEPServices', '1.0.0'
```
Comment (sbenedicadb) [inline Documentation/Migration/Migration_Install.md:23]: are we bundling the core extensions with the AEPCore cocoapod?  also should be version `3.0.0`

Comment (nporter-adbe) [reply Documentation/Migration/Migration_Install.md:23]: As of now, we are not. If we decide to bundle them we can update this doc.

File: Documentation/Migration/Migration_Install.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+### Manual Installation
+If Cocoapods is not an option, the compatibility layer can be built manually using the following commands:
+
+```bash
+git clone https://github.com/adobe/aep-sdk-compatibility-ios.git
+cd aep-sdk-compatibility
+make make-libs
+```
+After running these commands, the `.xcframework's` can be found within the build directory. Drag and drop these into an Xcode project and ensure that "copy items if needed" is checked.
```
Comment (sbenedicadb) [inline Documentation/Migration/Migration_Install.md:39]: `.xcframework`s

File: Documentation/Migration/Migration_Install.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## Sample Apps
+
+To download more examples of integrating the AEP Swift SDK via the Compatibility layer, head over to our sample apps.
```
Comment (sbenedicadb) [inline Documentation/Migration/Migration_Install.md:43]: from here to the bottom of the document, remove `our/your` references

**Review by sbenedicadb — CHANGES_REQUESTED** [review]
please see comments :)

File: Documentation/Migration/Migration_Install.md (hunk 5 above)
Comment (nporter-adbe) [reply Documentation/Migration/Migration_Install.md:39]: don't believe either is more correct AFAIK.

Comment (sbenedicadb) [reply Documentation/Migration/Migration_Install.md:39]: well for sure there's no apostrophe 😛

**Pull request merged**